string command
string filename
int32 camera_number
float32 zoom
bool write_file
